linker.ld

```diff
@@ -147,6 +147,7 @@ SECTIONS {
                 *(.teardown)
                 __teardown_end__ = .;
                 __bss_end__ = .;
+                end = __bss_end__;
```

Comment:
Where is this label needed? Why is there no _start_ label, but just an end label?